### PR TITLE
more: split util-linux and POSIX implementations

### PR DESCRIPTION
**Pull request by vitorhcl**
For #9612 and #11827.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

I don't think BSD platforms and osx need a platform-specific page, as there are many examples already.

Unfortunately, util-linux options are quite incompatible with POSIX, as well as some options on NetBSD and FreeBSD (and thus macOS), given that `less` and `more` share the same code on them. OpenBSD, as expected, fully follows POSIX, with the exception of the skip forward functionality, which is not currently implemented.

### CODE REVIEW

File: pages/common/more.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
 # more
 
-> Open a file for interactive reading, allowing scrolling and search.
-> More information: <https://manned.org/more>.
+> Interactively read a file, allowing scrolling and search.
```
Comment (gutjuri) [inline pages/common/more.md:3]:
```suggestion
> Interactively display a file, allowing scrolling and search.
```

File: pages/linux/more.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# more
+
+> Interactively read a file, allowing scrolling and search.
```
Comment (gutjuri) [inline pages/linux/more.md:3]:
```suggestion
> Interactively display a file, allowing scrolling and searching.
```

File: pages/linux/more.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+> See also: `less`.
+> More information: <https://manned.org/more>.
+
+- Open a file:
+
+`more {{path/to/file}}`
+
+- Open a file displaying from a specific line:
```
Comment (gutjuri) [inline pages/linux/more.md:11]:
```suggestion
- Display a specific line:
```

File: pages/common/more.md [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
 
 - Open a file:
 
 `more {{path/to/file}}`
 
+- Search case-[i]nsensitively when pressing "/":
+
+`more -i {{path/to/file}}`
+
 - Open a file displaying from a specific line:
```
Comment (gutjuri) [inline pages/common/more.md:15]:
```suggestion
- Display a specific line:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions.

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

LGTM, after @gutjuri's suggestions.

File: pages/common/more.md [hunk 5]
```diff
@@ -1,13 +1,18 @@
...
+> Interactively display a file, allowing scrolling and searching.
+> See also: `less`.
+> More information: <https://manned.org/more.1p>.
 
 - Open a file:
 
 `more {{path/to/file}}`
 
-- Open a file displaying from a specific line:
+- Search case-[i]nsensitively when pressing "/":
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/more.md:11]: IMO case-indensitively is correct

[the remark this answers is not in the document]
Comment (vitorhcl) [reply pages/common/more.md:11]: I agree with @gutjuri, what do you think @kbdharun?

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/more.md:11]: > I agree with @gutjuri, what do you think @kbdharun?

Same here, we can leave it as it is.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**